### PR TITLE
fixed closure-table failing tests

### PR DESCRIPTION
**Pull request by Salimlou**
Hi all,

this pull request makes the tests 'closure-table' goes green.

For review!

### CODE REVIEW

**Comment by Salimlou** [comment]
hmmm.... postgres failing, I'll try to fix it!

File: src/persistence/SubjectOperationExecutor.ts [hunk 1]
```diff
@@ -51,8 +51,8 @@ export class SubjectOperationExecutor {
     // -------------------------------------------------------------------------
 
     constructor(protected connection: Connection,
-                protected transactionEntityManager: EntityManager,
-                protected queryRunner: QueryRunner) {
+        protected transactionEntityManager: EntityManager,
+        protected queryRunner: QueryRunner) {
```
Comment (pleerock) [inline src/persistence/SubjectOperationExecutor.ts]: please don't reformat exist code

File: src/persistence/SubjectOperationExecutor.ts [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+import { Connection } from "../connection/Connection";
+import { QueryRunner } from "../query-runner/QueryRunner";
+import { JunctionInsert, JunctionRemove, Subject } from "./Subject";
+import { OrmUtils } from "../util/OrmUtils";
+import { EntityManager } from "../entity-manager/EntityManager";
+import { PromiseUtils } from "../util/PromiseUtils";
+import { MongoDriver } from "../driver/mongodb/MongoDriver";
+import { ColumnMetadata } from "../metadata/ColumnMetadata";
+import { EmbeddedMetadata } from "../metadata/EmbeddedMetadata";
+import { Broadcaster } from "../subscriber/Broadcaster";
```
Comment (pleerock) [inline src/persistence/SubjectOperationExecutor.ts]: please don't reformat exist code

File: src/metadata-builder/ClosureJunctionEntityMetadataBuilder.ts [hunk 3]
```diff
@@ -57,7 +57,7 @@ export class ClosureJunctionEntityMetadataBuilder {
                 args: {
                     target: "",
                     mode: "virtual",
-                    propertyName: "descendant_" + primaryColumn.databaseName,
+                    propertyName: "descendant",
```
Comment (pleerock) [inline src/metadata-builder/ClosureJunctionEntityMetadataBuilder.ts]: I can merge this test however its not absolutely correct. Closure tables are not working with multiple primary keys and thats a problem they have right now

Comment (pleerock) [inline src/metadata-builder/ClosureJunctionEntityMetadataBuilder.ts]: but maybe its okay as a temporary fix

**Comment by pleerock** [comment]
please upstream it with latest master changes and fix conflicts, thanks!

File: src/metadata-builder/ClosureJunctionEntityMetadataBuilder.ts (hunk 3 above)
Comment (Salimlou) [inline src/metadata-builder/ClosureJunctionEntityMetadataBuilder.ts]: I know it's temporary , it was just for having the basics working so we can use it!
I'll resolve the conflict ASAP.

File: src/persistence/SubjectOperationExecutor.ts (hunk 1 above)
Comment (Salimlou) [inline src/persistence/SubjectOperationExecutor.ts]: yeah sorry about that, I have automatic formatting enabled in vscode :)

**Pull request merged**